CHANGELOG.md

```diff
@@ -1,9 +1,13 @@
+## 0.5.1 (February 24, 2022)
+* **(Enterprise Only)** Regression with Terraform Cloud driver's `required_providers` configuration which leads the driver to potentially attempt to retrieve an incorrect provider source and version [[GH-728](https://github.com/hashicorp/consul-terraform-sync/issues/728)]
```

Comment:
Could you put `BUG FIXES:` above this?

Reply:
Yes! Totally missed that. Thank you for catching